Server/routes/authRoute.js

```diff
@@ -8,12 +8,14 @@ const {
   recoveryRequestController,
   validateRecoveryTokenController,
   resetPasswordController,
+  deleteUserController
 } = require("../controllers/authController");
 
 //Auth routes
 router.post("/register", registerController);
 router.post("/login", loginController);
 router.post("/user/:userId", verifyJWT, userEditController);
+router.delete("/delete/:userId", deleteUserController);
```

Comment:
This route should be `/user/:userId` so that it is clear what resource is being deleted and to be consistent with the edit endpoint.  We also don't need to specify `delete` in the route since it is clear from the HTTP operation `delete` that this is a delete operation.